LSP-copilot.sublime-settings

```diff
@@ -16,7 +16,8 @@
 		"local_checks": false,
 		"telemetry": false,
 		"proxy": "",
-		"completion_style": "popup"
+		"completion_style": "popup",
+		"respect_gitignore": false
```

Comment:
I would add a comment here and in the `sublime-package.json`'s `markdownDescription` section that the lookup is top level only

Reply:
I'll be pushing a change shortly that should have it functioning pretty closely to gitignore spec